benchmarking/nursery/benchmark_multiobjective/benchmark_definitions.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from benchmarking.commons.benchmark_definitions import (
+    SurrogateBenchmarkDefinition,
+    fcnet_benchmark_definitions,
+)
+
+
+def nas201_mo_benchmark(dataset_name):
+    return SurrogateBenchmarkDefinition(
+        max_wallclock_time=6 * 3600,
+        n_workers=4,
+        elapsed_time_attr="metric_elapsed_time",
+        metric=["metric_valid_error", "metric_params"],
```

Comment:
`metric` and `mode` are not supposed to be lists in this type.

Let us be a bit more careful here, and change some things in the def of `SurrogateBenchmarkDefinition`, for example the types becoming union's, and also add a check to `__post_init__` that the lists are the same size, etc.

Reply:
Thanks!